src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,43 @@
+package duke;
+/**
+ * Deadlines is a task that have a deadline which is a string.
+ * @author Dominic Siew Zhen Yu
+ *
+ */
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task{
```

Comment:
You might want to change the class name to `Deadline` instead since plural forms are usually for a collection of data such as an array of deadlines. 😄